python/spec/dependabot/python/file_parser_spec.rb

```diff
@@ -65,6 +65,29 @@
       its(:length) { is_expected.to eq(3) }
     end
 
+    context "with jinja templates" do
+      let(:requirements_fixture_name) { "jinja_requirements.txt" }
+      describe "the first dependency" do
+        subject(:dependency) { dependencies.first }
+
+        it "has the right details" do
+          dependency = dependencies.first
+          expect(dependency).to be_a(Dependabot::Dependency)
+          expect(dependency.name).to eq("psycopg2")
```

Comment:
Another way of writing this with the subject you have is replace the `it` block with this:

```
its(:name) { is_expected.to eq("psycopg2") }
its(:version) { is_expected.to eq("2.6.1") }
```